core/templates/dev/head/base.html

```diff
@@ -225,10 +225,44 @@
                             </ul>
                           </li>
                         {% endif %}
+                        {% if nav_mode != 'create' and nav_mode != 'explore' %}
+                          <create-exploration-button ng-if-start="windowWidth > 768"></create-exploration-button>
```

Comment:
Still not keen. Why is the div affecting the styling?

If we can't fix that by modifying oppia.css in a reasonable way, then instead of using ng-if-start let's put ng-ifs on each of the four outermost li's.